feddlib/core/Mesh/MeshPartitioner_def.hpp

```diff
@@ -247,11 +250,13 @@ void MeshPartitioner<SC,LO,GO,NO>::readAndPartitionMesh( int meshNumber ){
     bool buildEdges = pList_->get("Build Edge List", true);
     bool buildSurfaces = pList_->get("Build Surface List", true);
 
+	// Adding surface as subelement to elements
```

Comment:
For our todo list: It might make sense to call them surface elements (even if somewhere in the interior of the domain) instead of subelements.